scripts/prepare.sh

```diff
@@ -37,7 +37,7 @@ clone_project()
 {
   if [ ! -d "${PLATFORM_PATH}/$2" ]; then
     printf "Clone %s\n" "$1"
-    git clone "$1" "${PLATFORM_PATH}/$2"
+    git clone "$1" "../$2"
```

Comment:
Please don't do this as it will make prepare.sh depend on the current working directory (again). Is there any problem with the `${PLATFORM_PATH}`?

Reply:
Fixed